bindings/netstandard/ElectionGuard/ElectionGuard.Decryption.Tests/Challenge/TestChallenge.cs

```diff
@@ -0,0 +1,146 @@
+
+using ElectionGuard.Encryption.Utils.Generators;
+using ElectionGuard.ElectionSetup.Tests.Generators;
+using ElectionGuard.Decryption.Decryption;
+using ElectionGuard.Decryption.Shares;
+using ElectionGuard.Decryption.ChallengeResponse;
+using ElectionGuard.Decryption.Extensions;
+using ElectionGuard.Decryption.Accumulation;
+using ElectionGuard.Decryption.Challenge;
+using ElectionGuard.Decryption.Tests.Decryption;
+
+namespace ElectionGuard.Decryption.Tests.Challenge;
+
+// Simple tests using the programmatically generated fake manifest
+[TestFixture]
+public class TestChallenge : DisposableBase
+{
+    private const int BALLOT_COUNT_CAST = 1;
+    private const int BALLOT_COUNT_CHALLENGED = 0;
+    private const int BALLOT_COUNT_SPOILED = 0;
+    private const int NUMBER_OF_GUARDIANS = 3;
+    private const int QUORUM = 2;
+
+    [Test] // Category("SingleTest")
+    public void Test_Challenge_Selection()
```

Comment:
this test serves as a step by step of the challenge process for a single selection and a single guardian